### DISCUSSION TITLE
Enabling Managed Mode still bricks the Node?

### DISCUSSION THREAD

**Discussion by SimbimChimbetov** [meshtastic/Discussions#60] [nRF52]
After enabling the Managed Mode the Nodes got unresponsive.
There was a Workaround a time ago but I can't find anything if it is fixed or not?

**Comment by rcarteraz** [comment]
> After enabling the Managed Mode the Nodes got unresponsive. There was a Workaround a time ago but I can't find anything if it is fixed or not?

Can you provide more information that is needed to potentially help you? What does "unresponsive" mean in this case? What hardware? What firmware version? What client are you trying to use? Do you have relevant log output from when the devices get unresponsive?

**Reply by SimbimChimbetov** [reply]
There was a known Problem with Rak4631 / 2.1.x Firmware ( https://www.youtube.com/watch?v=lJDaMv8n2jw )
 when you Enabled the Managed Mode and with it Bluetooth, the Rak got unresponsive, this means it stopped broadcasting or turned completely off.
Now I updated my Routers to 2.5.11 and wondering if this Problem still exist.

I don't dare to test because my Devices are really hard to access.

**Comment by SigIntRx** [comment]
i made the mistake of selecting managed mode on my T-deck and it's bricked now. can't flash, reboot, hard reset or anything. It's stuck on bluetooth programming mode. I just got it yesterday. 2.7.15.567beta.

**Reply by SimbimChimbetov** [reply]
Good to know that Managed Mode is still a Problem. 
You can try emptying the battery and then restart?

**Reply by thebentern** [reply]
You are still able to do a full erase and flash. This is controlled by the bootloader with the RST + hold BOOT process to open USB/JTAG mode.

**Reply by net-flyer** [reply]
> You are still able to do a full erase and flash. This is controlled by the bootloader with the RST + hold BOOT process to open USB/JTAG mode.

Exactly how do I do the above process? Thanks

**Reply by madeofstown** [reply]
> > You are still able to do a full erase and flash. This is controlled by the bootloader with the RST + hold BOOT process to open USB/JTAG mode.
> 
> Exactly how do I do the above process? Thanks

Press and hold the reboot button. While holding the reboot button press the trackball and release, then release the reset button.

**Comment by BugZappa** [comment]
Welp wish i knew about this, the description is really misleading. Bricked my Thinknode M5 in less than 24 hours. hah
